src/overlays/actors/ovl_Obj_Switch/z_obj_switch.h

```diff
@@ -17,21 +17,21 @@ typedef enum {
 } ObjSwitchType;
 
 typedef enum {
-    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_0,
-    /* 1 */ OBJSWITCH_SUBTYPE_FLOOR_1,
-    /* 2 */ OBJSWITCH_SUBTYPE_FLOOR_2,
-    /* 3 */ OBJSWITCH_SUBTYPE_FLOOR_3
+    /* 0 */ OBJSWITCH_SUBTYPE_FLOOR_NORMAL,     // Normal Yellow Switch
```

Comment:
all "NORMAL" subtypes should probably be more explicit about the behavior, both in the name and comment
(what is "normal" supposed to be?)

Reply:
Normal is the classic behavior of a switch, you turn it on and it'll set the switch flag, but that's just what it does

I wrote it that way because I thought people would understand what's each subtype easier, that's also why I mentioned the colors